pkgs/applications/misc/kiwix/lib.nix

```diff
@@ -51,4 +51,12 @@ stdenv.mkDerivation rec {
   postPatch = ''
     patchShebangs scripts
   '';
+
+  meta = with lib; {
+    description = "Common code base for all Kiwix ports";
+    homepage = "https://kiwix.org";
+    license = licenses.gpl3Plus;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ ajs124 colinsane ];
```

Comment:
I don't actually maintain any of this

Reply:
to be clear: you're listed in tree as the maintainer for the `kiwix` package in <pkgs/applications/misc/kiwix/default.nix>, and <pkgs/applications/misc/kiwix/lib.nix> is (was) sort of just an internal detail of that package.

i've updated this PR so that i'll take exclusive maintainership of the `lib.nix` file though, and i'll make sure the diff doesn't add your name to any lines it wasn't on before. **if** you're wanting to not be maintainer of the `kiwix` package either though, probably makes sense to open another PR just for that so that it's not tied up by this one.

Reply:
right. It's just that I don't actively use this software and therefore don't look after it. If you do, I'd appreciate you replacing me in meta.maintainers. I can still review and merge them, if nobody else does though.